test/e2e/pages/vl-tooltip.page.js

```diff
@@ -54,6 +53,15 @@ class VlTooltipPage extends Page {
     async load() {
         await super.load(Config.baseUrl + '/demo/vl-tooltip.html');
     }
+
+    async hideAllTooltips() {
+        const title = await this._getTitle();
+        await title.hover();
+    }
```

Comment:
hier zou ik een andere implementatie doen, misschien gewoon body.click() of zo?

Reply:
De body click doet het niet...

Reply:
Body hover wel precies.